src/plugins/dashboard/server/plugin.ts

```diff
@@ -83,9 +83,9 @@ export class DashboardPlugin
     this.logger.debug('dashboard: Started');
 
     if (plugins.taskManager) {
-      scheduleDashboardTelemetry(this.logger, plugins.taskManager);
-
-      plugins.taskManager.runNow(TASK_ID);
+      scheduleDashboardTelemetry(this.logger, plugins.taskManager)!.then(() => {
+        plugins.taskManager.runSoon(TASK_ID);
+      });
```

Comment:
Should we add a `.catch` to handle any errors thrown by `runSoon`. Can we also add a test for this?

Reply:
probably `scheduleDashboardTelemetry.then().catch()` right? That should catch errors in both? Otherwise we might see unhandled promise rejections